README.md

```diff
@@ -15,17 +17,19 @@ This is based on portions of [ThunderboltPatcher](https://github.com/osy/Thunder
 
 Thanks to t8012.dev and mrarm for assistance with the VDM and Ace2 host interface commands.
 
+The documentation of the ACE and its USB-PD VPDs is here https://blog.t8012.dev/ace-part-1/
+
 ## Building
 
 Install the XCode commandline tools and type `make`.
 
 ## Usage
 
-Connect the two devices via their DFU ports. That's the rear port on MacBooks and the port nearest to the power plug on Mac Minis.
+Connect the two devices via their DFU ports. That's the furthest left port on M1 MacBooks, the closest left port on the T2, and the only port on the iPad.  See Apple's DFU recovery support articles to identify these ports on other models.
 
-You need to use a *USB 3.0 compatible* (SuperSpeed) Type C cable. USB 2.0-only cables, including most cables meant for charging, will not work, as they do not have the required pins. Thunderbolt cables work too.
+You need to use a *USB 3.0 compatible* (SuperSpeed) Type C cable. USB 2.0-only cables, including most cables meant for charging, will not work, as they do not have the required pins (USB CC1/CC2 where USB-PD are transmitted). Thunderbolt cables work too.
```

Comment:
This is not correct; the missing pins are SBU1/SBU2, which we use for serial. USB 2.0 cables *do* work for USB-PD commands like reboot, just not for serial.

Reply:
Honestly its missing both... but neither here nor there (it only ends up connecting D+/D-  SS1/2 and VCC / GND on a 2.0 cable)